### DISCUSSION TITLE
Part 3

### DISCUSSION THREAD

**Discussion by LayZeeDK** [this-is-learning/rxjs-fundamentals-course#3] [Content contributions]
Discussing and coordinating [Part 3](https://github.com/this-is-learning/rxjs-fundamentals-course/blob/main/part-3/part-3.md) efforts.

**Comment by LayZeeDK** [comment]
This part is currently in Russian. @Armenvardanyan95 will translate it to English.

**Reply by LayZeeDK** [reply]
https://github.com/this-is-learning/rxjs-fundamentals-course/discussions/3#discussioncomment-501941

**Reply by LayZeeDK** [reply]
Open PR for the translation to English: https://github.com/this-is-learning/rxjs-fundamentals-course/pull/7

**Comment by Armenvardanyan95** [comment]
I'm in on it right now

**Reply by LayZeeDK** [reply]
https://github.com/this-is-learning/rxjs-fundamentals-course/discussions/3#discussioncomment-501940

**Comment by LayZeeDK** [comment]
# Content review
The content of this part needs review.

**Comment by LayZeeDK** [comment]
# Code review
The code snippets in this part need review.

**Reply by LayZeeDK** [reply]
See https://github.com/this-is-learning/rxjs-fundamentals-course/pull/7#discussion_r597977106 for a suggestion by @prawin-s.

**Comment by LayZeeDK** [comment]
# Error handling
Improve the esction about error handling to include:
- `throwError`
- `catchError`
- `throw`ing

**Comment by LayZeeDK** [comment]
# Timers and intervals
Review the sections about the `timer` and `interval` operators. Figure out how to express that `timer` is like `setTimeout` followed by `setInterval` when it's passed 2 arguments since it adds an initial delay as specified by the first argument. And that `interval` is like `setInterval`.

**Comment by LayZeeDK** [comment]
# Contributors
@Armenvardanyan95 is actively working on this part.

**Comment by LayZeeDK** [comment]
See suggestion at https://github.com/this-is-learning/rxjs-fundamentals-course/pull/12/files/091e574113fc9f038d88810d4af897f295089518#diff-9c2bc935db665401110e43ee7ee480ec2343dadebbce51e80be9542e398707bc

**Reply by Armenvardanyan95** [reply]
@LayZeeDK somehow this link won't open the suggestion itself for me. Which one do you mean?

**Reply by LayZeeDK** [reply]
![image](https://user-images.githubusercontent.com/6364586/131258504-cd277238-d779-4bb6-a837-b2cadf9898e9.png)

https://github.com/this-is-learning/rxjs-fundamentals-course/pull/12#discussion_r602295493
